scene/gui/texture_progress.cpp

```diff
@@ -263,6 +264,7 @@ void TextureProgress::draw_nine_patch_stretched(const Ref<Texture> &p_texture, F
 				dst_rect.size.y = width_filled;
 				topleft.y = last_section_size;
 			} break;
+			default: {}
```

Comment:
Both cases:
```
scene/gui/texture_progress.cpp:217:10: warning: enumeration value 'FILL_BILINEAR_LEFT_AND_RIGHT' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:217:10: warning: enumeration value 'FILL_BILINEAR_TOP_AND_BOTTOM' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:217:10: warning: enumeration value 'FILL_CLOCKWISE_AND_COUNTER_CLOCKWISE' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:217:10: warning: enumeration value 'FILL_CLOCKWISE' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:217:10: warning: enumeration value 'FILL_COUNTER_CLOCKWISE' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:241:10: warning: enumeration value 'FILL_BILINEAR_LEFT_AND_RIGHT' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:241:10: warning: enumeration value 'FILL_BILINEAR_TOP_AND_BOTTOM' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:241:10: warning: enumeration value 'FILL_CLOCKWISE_AND_COUNTER_CLOCKWISE' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:241:10: warning: enumeration value 'FILL_CLOCKWISE' not handled in switch [-Wswitch]
scene/gui/texture_progress.cpp:241:10: warning: enumeration value 'FILL_COUNTER_CLOCKWISE' not handled in switch [-Wswitch]
```
@MednauN (1a2622d344)

Reply:
When I was writing this code, there were no `FILL_BILINEAR_LEFT_AND_RIGHT` and `FILL_BILINEAR_TOP_AND_BOTTOM` modes.
I think, they should be implemented (someday), so at least a `// TODO` comment will be appreciated here.

And I believe, `FILL_CLOCKWISE` and the rest can be ignored here. Because they are made with circle-shaped textures in mind and there is a little meaning in drawing them as 9-patches.

Reply:
CC @Calinou who implemented the new fill options.

Reply:
Thanks for the feedback btw, I've added `TODO`s and a clear explanation about circular shapes in the next commit.